tests/nevermind_test.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+from plugins.nevermind import Nevermind
+
+pytest_plugins = ['errbot.backends.test']
+
+extra_plugin_dir = '../plugins'
+
+def test_nevermind(testbot):
+    testbot.push_message('!nevermind')
+    assert "I'm sorry :(" in testbot.pop_message()
```

Comment:
Sorry, I was thinking of a class based test suite, but it looks overkill, let's drop it.